CI fails because because of pyopencl deprecation warnings

CI fails with some deprecation warnings from pyopencl, e.g.:

```FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_projection.py::TestProj::test_proj - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).```

<details><summary>logs</summary>

```
self = <silx.opencl.test.test_projection.TestProj testMethod=test_proj>

    def setUp(self):
        if ocl is None:
            return
        # ~ if sys.platform.startswith('darwin'):
        # ~ self.skipTest("Projection is not implemented on CPU for OS X yet")
        self.getfiles()
        n_angles = self.sino.shape[0]
>       self.proj = projection.Projection(self.phantom.shape, n_angles)

/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_projection.py:60: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/projection.py:146: in __init__
    self._use_textures = self.check_textures_availability()
/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/processing.py:183: in check_textures_availability
    return check_textures_availability(self.ctx)
/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/common.py:779: in check_textures_availability
    dummy_texture = allocate_texture(ctx, (16, 16))
/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/common.py:762: in allocate_texture
    return pyopencl.Image(
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <pyopencl._cl.Image object at 0x7f3db26fcef0>
context = <pyopencl.Context at 0x5567d2c27488 on <pyopencl.Device 'AMD EPYC 7763 64-Core Processor' on 'Intel(R) OpenCL' at 0x5567d0c92368>>
flags = 12, format = ImageFormat(INTENSITY, FLOAT), shape = (16, 16)
pitches = None
hostbuf = array([[0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.],
       [0., 0., 0., 0., 0., 0., 0., 0., 0., 0...0., 0., 0., 0., 0., 0.],
       [0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.]],
      dtype=float32)
is_array = False, buffer = None

    def image_init(
                self, context, flags, format, shape=None, pitches=None,
                hostbuf=None, is_array=False, buffer=None, *,
                desc: ImageDescriptor | None = None,
                _through_create_image: bool = False,
            ) -> None:
        if hostbuf is not None and not \
                (flags & (mem_flags.USE_HOST_PTR | mem_flags.COPY_HOST_PTR)):
            warn("'hostbuf' was passed, but no memory flags to make use of it.",
                 stacklevel=2)
    
        if desc is not None:
            if shape is not None:
                raise TypeError("shape may not be passed when using descriptor")
            if pitches is not None:
                raise TypeError("pitches may not be passed when using descriptor")
            if is_array:
                raise TypeError("is_array may not be passed when using descriptor")
            if buffer is not None:
                raise TypeError("is_array may not be passed when using descriptor")
    
            Image._custom_init(self, context, flags, format, desc, hostbuf)
    
            return
    
        if shape is None and hostbuf is None:
            raise Error("'shape' must be passed if 'hostbuf' is not given")
    
        if shape is None and hostbuf is not None:
            shape = hostbuf.shape
    
        if hostbuf is None and pitches is not None:
            raise Error("'pitches' may only be given if 'hostbuf' is given")
    
        if context._get_cl_version() >= (1, 2) and get_cl_header_version() >= (1, 2):
            if not _through_create_image:
>               warn("Non-descriptor Image constructor called. "
                     "This will stop working in 2026. "
                     "Use create_image instead (with the same arguments).",
                     DeprecationWarning, stacklevel=2)
E               DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).

/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/pyopencl/__init__.py:928: DeprecationWarning
=========================== short test summary info ============================
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_backprojection.py::TestFBP::test_fbp - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_backprojection.py::TestFBP::test_fbp_filters - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_backprojection.py::TestFBP::test_fbp_oddsize - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_1D[convolution_data0] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_separable_2D[convolution_data0] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_separable_3D[convolution_data0] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_nonseparable_2D[convolution_data0] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_nonseparable_3D[convolution_data0] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_batched_2D[convolution_data0] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_1D[convolution_data1] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_separable_2D[convolution_data1] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_separable_3D[convolution_data1] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_nonseparable_2D[convolution_data1] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_nonseparable_3D[convolution_data1] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_batched_2D[convolution_data1] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_1D[convolution_data2] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_separable_2D[convolution_data2] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_separable_3D[convolution_data2] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_nonseparable_2D[convolution_data2] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_nonseparable_3D[convolution_data2] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_batched_2D[convolution_data2] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_1D[convolution_data3] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_separable_2D[convolution_data3] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_separable_3D[convolution_data3] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_nonseparable_2D[convolution_data3] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_nonseparable_3D[convolution_data3] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_batched_2D[convolution_data3] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_1D[convolution_data4] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_separable_2D[convolution_data4] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_separable_3D[convolution_data4] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_nonseparable_2D[convolution_data4] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_nonseparable_3D[convolution_data4] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_batched_2D[convolution_data4] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_1D[convolution_data5] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_separable_2D[convolution_data5] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_separable_3D[convolution_data5] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_nonseparable_2D[convolution_data5] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_nonseparable_3D[convolution_data5] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_batched_2D[convolution_data5] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_1D[convolution_data6] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_separable_2D[convolution_data6] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_separable_3D[convolution_data6] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_nonseparable_2D[convolution_data6] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_nonseparable_3D[convolution_data6] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_batched_2D[convolution_data6] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_1D[convolution_data7] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_separable_2D[convolution_data7] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_separable_3D[convolution_data7] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_nonseparable_2D[convolution_data7] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_nonseparable_3D[convolution_data7] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_batched_2D[convolution_data7] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_1D[convolution_data8] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_separable_2D[convolution_data8] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_separable_3D[convolution_data8] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_nonseparable_2D[convolution_data8] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_nonseparable_3D[convolution_data8] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_batched_2D[convolution_data8] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_1D[convolution_data9] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_separable_2D[convolution_data9] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_separable_3D[convolution_data9] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_nonseparable_2D[convolution_data9] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_nonseparable_3D[convolution_data9] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_batched_2D[convolution_data9] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_1D[convolution_data10] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_separable_2D[convolution_data10] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_separable_3D[convolution_data10] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_nonseparable_2D[convolution_data10] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_nonseparable_3D[convolution_data10] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_batched_2D[convolution_data10] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_1D[convolution_data11] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_separable_2D[convolution_data11] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_separable_3D[convolution_data11] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_nonseparable_2D[convolution_data11] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_nonseparable_3D[convolution_data11] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_batched_2D[convolution_data11] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_1D[convolution_data12] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_separable_2D[convolution_data12] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_separable_3D[convolution_data12] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_nonseparable_2D[convolution_data12] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_nonseparable_3D[convolution_data12] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_batched_2D[convolution_data12] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_1D[convolution_data13] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_separable_2D[convolution_data13] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_separable_3D[convolution_data13] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_nonseparable_2D[convolution_data13] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_nonseparable_3D[convolution_data13] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_batched_2D[convolution_data13] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_1D[convolution_data14] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_separable_2D[convolution_data14] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_separable_3D[convolution_data14] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_nonseparable_2D[convolution_data14] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_nonseparable_3D[convolution_data14] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_batched_2D[convolution_data14] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_1D[convolution_data15] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_separable_2D[convolution_data15] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_separable_3D[convolution_data15] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_nonseparable_2D[convolution_data15] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_nonseparable_3D[convolution_data15] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_batched_2D[convolution_data15] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_1D[convolution_data16] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_separable_2D[convolution_data16] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_separable_3D[convolution_data16] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_nonseparable_2D[convolution_data16] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_nonseparable_3D[convolution_data16] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_batched_2D[convolution_data16] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_1D[convolution_data17] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_separable_2D[convolution_data17] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_separable_3D[convolution_data17] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_nonseparable_2D[convolution_data17] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_nonseparable_3D[convolution_data17] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_batched_2D[convolution_data17] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_1D[convolution_data18] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_separable_2D[convolution_data18] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_separable_3D[convolution_data18] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_nonseparable_2D[convolution_data18] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_nonseparable_3D[convolution_data18] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_batched_2D[convolution_data18] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_1D[convolution_data19] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_separable_2D[convolution_data19] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_separable_3D[convolution_data19] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_nonseparable_2D[convolution_data19] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_nonseparable_3D[convolution_data19] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_batched_2D[convolution_data19] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_1D[convolution_data20] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_separable_2D[convolution_data20] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_separable_3D[convolution_data20] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_nonseparable_2D[convolution_data20] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_nonseparable_3D[convolution_data20] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_batched_2D[convolution_data20] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_1D[convolution_data21] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_separable_2D[convolution_data21] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_separable_3D[convolution_data21] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_nonseparable_2D[convolution_data21] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_nonseparable_3D[convolution_data21] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_batched_2D[convolution_data21] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_1D[convolution_data22] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_separable_2D[convolution_data22] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_separable_3D[convolution_data22] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_nonseparable_2D[convolution_data22] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_nonseparable_3D[convolution_data22] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_batched_2D[convolution_data22] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_1D[convolution_data23] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_separable_2D[convolution_data23] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_separable_3D[convolution_data23] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_nonseparable_2D[convolution_data23] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_nonseparable_3D[convolution_data23] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_batched_2D[convolution_data23] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_1D[convolution_data24] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_separable_2D[convolution_data24] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_separable_3D[convolution_data24] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_nonseparable_2D[convolution_data24] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_nonseparable_3D[convolution_data24] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_batched_2D[convolution_data24] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_1D[convolution_data25] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_separable_2D[convolution_data25] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_separable_3D[convolution_data25] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_nonseparable_2D[convolution_data25] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_nonseparable_3D[convolution_data25] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_batched_2D[convolution_data25] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_1D[convolution_data26] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_separable_2D[convolution_data26] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_separable_3D[convolution_data26] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_nonseparable_2D[convolution_data26] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_nonseparable_3D[convolution_data26] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_batched_2D[convolution_data26] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_1D[convolution_data27] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_separable_2D[convolution_data27] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_separable_3D[convolution_data27] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_nonseparable_2D[convolution_data27] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_nonseparable_3D[convolution_data27] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_convolution.py::test_batched_2D[convolution_data27] - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
FAILED ../../../../../opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/silx/opencl/test/test_projection.py::TestProj::test_proj - DeprecationWarning: Non-descriptor Image constructor called. This will stop working in 2026. Use create_image instead (with the same arguments).
```

</summary>


attn @kif @pierrepaleo

--- a/src/silx/opencl/common.py
+++ b/src/silx/opencl/common.py
@@ -38,6 +38,7 @@ __status__ = "stable"
 
 import os
 import logging
+from packaging.version import Version
 import numpy
 from .utils import get_opencl_code
 
@@ -757,17 +758,24 @@ def allocate_texture(ctx, shape, hostbuf=None, support_1D=False):
         do not support 1D images, so 1D images are handled as 2D with one row
     :param support_1D: force the image to be 1D if the shape has only one dim
     """
+    # TODO "shape" as optional parameter (kwarg) ?
     if len(shape) == 1 and not (support_1D):
         shape = (1,) + shape
-    return pyopencl.Image(
+    if hostbuf is None:
+        hostbuf = numpy.zeros(shape[::-1], dtype=numpy.float32)
+
+    if Version(pyopencl.version.VERSION_TEXT) >= Version("2024.3"):
+        texture_creation_function = pyopencl.create_image
+    else:
+        texture_creation_function = pyopencl.Image
+    return texture_creation_function(
         ctx,
         pyopencl.mem_flags.READ_ONLY | pyopencl.mem_flags.USE_HOST_PTR,
         pyopencl.ImageFormat(
             pyopencl.channel_order.INTENSITY, pyopencl.channel_type.FLOAT
         ),
-        hostbuf=numpy.zeros(shape[::-1], dtype=numpy.float32),
+        hostbuf=hostbuf,
     )
-
 
 def check_textures_availability(ctx):
     """

--- a/src/silx/opencl/projection.py
+++ b/src/silx/opencl/projection.py
@@ -222,14 +222,8 @@ class Projection(OpenclProcessing):
         self.add_to_cl_mem({"d_slice": ary})
 
     def allocate_textures(self):
-        self.d_image_tex = pyopencl.Image(
-            self.ctx,
-            mf.READ_ONLY | mf.USE_HOST_PTR,
-            pyopencl.ImageFormat(
-                pyopencl.channel_order.INTENSITY, pyopencl.channel_type.FLOAT
-            ),
-            hostbuf=np.ascontiguousarray(self._tmp_extended_img.T),
-        )
+        hostbuf = np.ascontiguousarray(self._tmp_extended_img.T)
+        self.d_image_tex = self.allocate_texture(hostbuf.shape, hostbuf)
 
     def transfer_to_texture(self, image):
         image2 = image

